server/blob/blob-objectstorage/src/test/java/org/apache/james/blob/objectstorage/crypto/CryptoConfigBuilderTest.java

```diff
@@ -0,0 +1,82 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.blob.objectstorage.crypto;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.crypto.tink.subtle.Hex;
+
+class CryptoConfigBuilderTest {
+
+    public static final char[] PASSWORD = "password".toCharArray();
+    public static final String SALT = "0123456789abcdef";
+
+    @Test
+    void should_not_build_crypto_config_if_salt_missing() {
```

Comment:
Why are we using **snake case**? I think we enforce the use of camel case

Reply:
sorry about that too much project switching. I have fixed it (though I believe there is an argument to be made about readability of the test cases being improved by snake case.)

Reply:
Many thanks!